docs/settingUp.md

```diff
@@ -32,12 +40,16 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
+   
+ **Verification** : Check the gradle folder for the existence of the SDK.
```

Comment:
Might need to mention where the gradle folder is (new devs may mistake it for the one found in the project root)